### ISSUE TITLE
Access to undeclared static property: Fragen\Git_Updater\Settings::$nonce

### ISSUE THREAD

**Issue by wilbertcaba** [afragen/git-updater#978]
Last time I used this plugin was last week with no issues. Today I pushed changes to my Github remote branch and made sure to manually change the style.css version number, then I received an email from WordPress with the following error that I can't get to troubleshoot:

```
Error Details
=============
An error of type E_ERROR was caused in line 63 of the file /nas/content/live/galowayjefcoat/wp-content/plugins/git-updater/src/Git_Updater/Settings.php. Error message: Uncaught Error: Access to undeclared static property: Fragen\Git_Updater\Settings::$nonce in /nas/content/live/galowayjefcoat/wp-content/plugins/git-updater/src/Git_Updater/Settings.php:63
Stack trace:
#0 /nas/content/live/galowayjefcoat/wp-content/plugins/git-updater/vendor/afragen/singleton/Singleton.php(44): Fragen\Git_Updater\Settings->__construct(NULL)
#1 /nas/content/live/galowayjefcoat/wp-content/plugins/git-updater/src/Git_Updater/Base.php(142): Fragen\Singleton::get_instance('Settings', Object(Fragen\Git_Updater\Base))
#2 /nas/content/live/galowayjefcoat/wp-includes/class-wp-hook.php(303): Fragen\Git_Updater\Base->load('')
#3 /nas/content/live/galowayjefcoat/wp-includes/class-wp-hook.php(327): WP_Hook->apply_filters(NULL, Array)
#4 /nas/content/live/galowayjefcoat/wp-includes/plugin.php(470): WP_Hook->do_action(Array)
#5 /nas/content/live/galowayjefcoat/wp-settings.php(578): do_action('init')
#6 /nas/content/live/galowayjefcoat/wp-config.php(120): require_once('/nas/content/li...')
#7 /nas/content/li
```

I assume this is the reason why I'm not able to see an available update for my theme after pushing the code to Github.

🆘  🙏🏼 

WordPress version 5.8.2
Current theme: Galloway Jefcoat (version 12.30.2021.2)
Current plugin: Git Updater (version 10.6.9)
PHP version 7.4.26

**Comment by afragen** [comment]
Odd as the `static $nonce` is declared in class GU_Trait which is used in Settings an should also bring in the variable to the Settings class. I'll try to look closer too.

**Comment by afragen** [comment]
I'm working my way through this. You can follow on the develop branch and see if that fixed it for you yet.

**Comment by afragen** [comment]
I think I've fixed all the nonce issues, can you check out the develop branch and let me know if it solves your issue?

Thanks.

**Comment by wilbertcaba** [comment]
@afragen I've been swamped with other tasks but will definitely test your update since I have to circle back to this task next week mandatorily and will keep you posted.

**Comment by afragen** [comment]
I released these changes in v10.6.10
